terraform/58-kubernetesagentworker/kubernetes_agent_workers.tf

```diff
@@ -0,0 +1,17 @@
+resource "octopusdeploy_kubernetes_agent_worker" "agent_with_minimum" {
+  name         = "minimum-agent"
+  worker_pool_ids = [octopusdeploy_static_worker_pool.workerpool_docker.id]
```

Comment:
Perhaps it's overkill, but it would be nice to set more than one worker pool here and assert that the created worker has all the right worker pool ids associated with it.

I do also acknowledge the fact that these integration tests are more or less a sanity check to see if it all 'compiles and runs', and not meant to be an exhaustive check of all the field mappings..